mycroft/skills/skill_loader.py

```diff
@@ -29,6 +34,49 @@
 SKILL_MAIN_MODULE = '__init__.py'
 
 
+def remove_submodule_refs(module_name):
+    """Ensure submodules being reloaded by removing the refs from sys.modules.
+
+    Python import system puts a reference for each module in the sys.modules
+    dictionary to bypass loading if a module is already in memory. To make
+    sure skills are completely reloaded these references are deleted.
+
+    Arguments:
+        module_name: name of skill module.
+    """
+    submodules = []
+    LOG.info('Skill module'.format(module_name))
```

Comment:
Is this intended to be an info level log message?

Reply:
You're right that should be a debug level, will fix on my train ride home

Reply:
Changed to debug